workbench-core/accounts/src/services/costCenterService.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { GetItemCommandOutput } from '@aws-sdk/client-dynamodb';
+import { AwsService, resourceTypeToKey, buildPkSk, uuidWithLowercasePrefix } from '@aws/workbench-core-base';
+import Boom from '@hapi/boom';
+import Account from '../models/account';
+import CostCenter from '../models/costCenter';
+import CreateCostCenter from '../models/createCostCenter';
+import AccountService from './accountService';
+
+export default class CostCenterService {
+  private _aws: AwsService;
+  private readonly _tableName: string;
+
+  public constructor(constants: { TABLE_NAME: string }) {
+    const { TABLE_NAME } = constants;
+    this._tableName = TABLE_NAME;
+    this._aws = new AwsService({ region: process.env.AWS_REGION!, ddbTableName: TABLE_NAME });
+  }
+
+  public async getCostCenter(costCenterId: string): Promise<CostCenter> {
+    // Get by id
+    const response = (await this._aws.helpers.ddb
+      .get(buildPkSk(costCenterId, resourceTypeToKey.costCenter))
+      .execute()) as GetItemCommandOutput;
+
+    if (response.Item === undefined) {
+      throw Boom.notFound(`Could not find cost center ${costCenterId}`);
+    }
+    const costCenter = response.Item as unknown as CostCenter;
+
+    return Promise.resolve(costCenter);
+  }
+
+  public async create(createCostCenter: CreateCostCenter): Promise<CostCenter> {
+    const id = uuidWithLowercasePrefix(resourceTypeToKey.costCenter);
+
+    const account = await this._getAccount(createCostCenter.accountId);
+
+    const createdAt = new Date(Date.now()).toISOString();
+
+    const costCenter: CostCenter = {
+      createdAt: createdAt,
+      updatedAt: createdAt,
+      id: id,
+      awsAccountId: account.awsAccountId,
```

Comment:
Can we add a comment to break up where account information begins? Let's move `name: createCostCenter.name` up to line 49 so it's grouped together with the other "native" costCenter info.